subt/main.py

```diff
@@ -896,71 +896,4 @@ def join(self, timeout=None):
         self.thread.join(timeout)
 
 
-def main():
-    import argparse
-    from osgar.lib.config import config_load
-    from osgar.record import record
-
-    parser = argparse.ArgumentParser(description='SubT Challenge')
```

Comment:
Here I see the problem that merge of this change breaks all SubT *real* robots, I mean they all will have to adapt their JSON configurations.

Reply:
Hmm, that is right. Not only we would have to update them all, but it would lead to increased duplication :worried:. Any ideas?

Reply:
Well, originally I had in mind extra JSON just with params, but they are closely coupled to the application ... so maybe module app + params ... but then there are the links ... i.e. I do not know yet.

Reply:
Yeah, back to the drawing board. Making it draft again.